misoc/targets/kasli.py

```diff
@@ -192,6 +192,7 @@ def __init__(self, *args, ethmac_nrxslots=2, ethmac_ntxslots=2, **kwargs):
 
         self.submodules.ethmac = LiteEthMAC(
                 phy=self.ethphy, dw=32, interface="wishbone",
+                endianness="little" if kwargs.get("cpu_type", "lm32") == "vexriscv" else "big",
```

Comment:
Other boards (KC705 in particular) also need this change.
You should be able to access the CPU type as ``self.cpu_type`` I believe, instead of duplicating this kind of code with kwargs.

Reply:
> You should be able to access the CPU type as self.cpu_type I believe

Yes. Will modify it in the next push.

> Other boards (KC705 in particular) also need this change.

I think Kasli, KC705, Metlino and Sayma (AMC) needs the change.
Should I add it to other targets that use `LiteEthMAC` as well? (namely `afc3v1.py`, `mlabs_video.py`, `simple.py`)

Reply:
>  Should I add it to other targets that use `LiteEthMAC` as well? (namely `afc3v1.py`, `mlabs_video.py`, `simple.py`)

Yes